packages/compass-collection/src/modules/collection-tab.ts

```diff
@@ -32,12 +33,7 @@ export type CollectionState = {
     | 'free_storage_size'
   > | null;
   metadata: CollectionMetadata | null;
-  currentTab:
-    | 'Documents'
-    | 'Aggregations'
-    | 'Schema'
-    | 'Indexes'
-    | 'Validation';
+  currentTab: CollectionSubtab;
```

Comment:
The source of truth for this is now workspaces plugin, so we should remove this from the store in the collection tab otherwise we are regressing back to duplicating workspaces state in the application

Reply:
Yes, I was removing this as part of communicating the (tab) change back to the workspaces.